src/pages/EnablePayments/OnfidoStep.js

```diff
@@ -28,60 +25,58 @@ const defaultProps = {
+
+    const goBack = useCallback(() => {
+        Wallet.updateCurrentStep(CONST.WALLET.STEP.ADDITIONAL_DETAILS);
+    }, []);
+
+    const reportError = useCallback(() => {
+        Growl.error(translate('onfidoStep.genericError'), 10_000);
+    }, [translate]);
 
-    render() {
-        return (
-            <>
-                <HeaderWithBackButton
-                    title={this.props.translate('onfidoStep.verifyIdentity')}
-                    onBackButtonPress={() => Wallet.updateCurrentStep(CONST.WALLET.STEP.ADDITIONAL_DETAILS)}
-                />
-                <FullPageOfflineBlockingView>
-                    {this.canShowOnfido() ? (
-                        <Onfido
-                            sdkToken={this.props.walletOnfidoData.sdkToken}
-                            onError={() => {
-                                Growl.error(this.props.translate('onfidoStep.genericError'), 10000);
-                            }}
-                            onUserExit={() => {
-                                Navigation.goBack();
-                            }}
-                            onSuccess={(data) => {
-                                BankAccounts.verifyIdentity({
-                                    onfidoData: JSON.stringify({
-                                        ...data,
-                                        applicantID: this.props.walletOnfidoData.applicantID,
-                                    }),
-                                });
-                            }}
-                        />
-                    ) : (
-                        <OnfidoPrivacy walletOnfidoData={this.props.walletOnfidoData} />
-                    )}
-                </FullPageOfflineBlockingView>
-            </>
-        );
-    }
+    const verifyIdentity = useCallback((data) => {
+        BankAccounts.verifyIdentity({
+            onfidoData: JSON.stringify({
+                ...data,
+                applicantID: walletOnfidoData.applicantID,
+            }),
+        });
+    }, [walletOnfidoData.applicantID]);
+
+    return (
+        <>
+            <HeaderWithBackButton
+                title={translate('onfidoStep.verifyIdentity')}
+                onBackButtonPress={goBack}
+            />
+            <FullPageOfflineBlockingView>
+                {shouldShowOnfido ? (
+                    <Onfido
+                        sdkToken={walletOnfidoData.sdkToken}
+                        onUserExit={Navigation.goBack}
+                        onError={reportError}
+                        onSuccess={verifyIdentity}
+                    />
+                ) : (
+                    <OnfidoPrivacy walletOnfidoData={walletOnfidoData} />
+                )}
+            </FullPageOfflineBlockingView>
+        </>
+    );
 }
 
 OnfidoStep.propTypes = propTypes;
 OnfidoStep.defaultProps = defaultProps;
```

Comment:
please add the `displayName` https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#function-component-style